### PR TITLE
First public release

### PR DESCRIPTION
**Pull request by tjwp**
The only file changed from the previous internal development is CHANGELOG.md.

### CODE REVIEW

**Comment by tjwp** [comment]
Reviewer: @justin-yesware

File: .ruby-gemset [hunk 1]
```diff
@@ -0,0 +1 @@
+yesware
```
Comment (justinmills) [inline .ruby-gemset]: Should we include this for the public version? Seems safe enough I suppose.

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+ruby-2.0.0-p598
```
Comment (justinmills) [inline .ruby-version]: Bump to ruby-2.2? Or maybe drop the patch level bits?

**Comment by justinmills** [comment]
I didn't read a ton of the ruby code here as I figured most was copied directly from the master version in the source repo. Mostly looked at configuration/comments. Couple of small comments, but nothing that needs to hold this up.

Reminds me that I should probably document how to use the HTTP client in the Readme.

File: .ruby-gemset (hunk 1 above)
Comment (tjwp) [inline .ruby-gemset]: Removed it

File: .ruby-version (hunk 2 above)
Comment (tjwp) [inline .ruby-version]: Updated to the latest ruby-2.2.3 since it was easy, tests ran cleanly.

**Comment by tjwp** [comment]
The only thing I had changed was the CHANGELOG. There's also an unnecessary ref to the currently private yeti_logger, which I've removed.

**Comment by justinmills** [comment]
g2g

**Comment by tjwp** [comment]
Squashed in place

**Pull request merged**